src/test/java/seedu/address/model/person/PersonTest.java

```diff
@@ -102,9 +102,9 @@ public void toStringMethod() {
                 + "{name=" + ALICE.getName()
                 + ", phone=" + ALICE.getPhone()
                 + ", email=" + ALICE.getEmail()
-                + ", room number=" + ALICE.getRoomNumber()
+                + ", room number=" + ALICE.getRoomNumber().orElse(null)
```

Comment:
Should either "room number" be changed to "roomNumber" or "emergencyContact" be changed to "emergency contact" for standardization?